core/exceptions/ExceptionStackTraceDisplay.php

```diff
@@ -68,73 +68,75 @@ protected function displayException(Exception $exception)
             ? $this->developerError($exception, $msg, $code, $trace_details)
             : $this->genericError($msg, $code);
         // start gathering output
-        $output = '
-<div id="ee-error-message" class="error">
-    ' . $error_message . '
-</div>';
-        $styles = $this->exceptionStyles();
+        $output = '<div id="ee-error-message" class="error">
+            ' . $error_message . '
+        </div>';
         $scripts = $this->printScripts(true);
         if (defined('DOING_AJAX')) {
-            echo wp_json_encode(array('error' => $styles . $output . $scripts));
+            echo wp_json_encode(array('error' => $output . $scripts));
             exit();
         }
-        echo $styles, $output, $scripts;
+        echo wp_kses($output . $scripts, AllowedTags::getWithScriptAndStyleTags());
     }
 
 
     private function genericError($msg, $code)
     {
-        return '
-    <p>
-        <span class="ee-error-user-msg-spn">' . trim($msg) . '</span> &nbsp; <sup>' . $code . '</sup>
-    </p>';
+        return '<p>
+            <span class="ee-error-user-msg-spn">' . trim($msg) . '</span> &nbsp; <sup>' . $code . '</sup>
+        </p>';
```

Comment:
same thing here, plz keep newlines in any formatted markup like this...

OR

extract to a separate template file (but maintain formatting there as well!)